deployment/network-operator/README.md

```diff
@@ -453,6 +453,18 @@ Specifies components to deploy in order to facilitate a secondary network in Kub
 | `ipamPlugin.repository` | string | `ghcr.io/k8snetworkplumbingwg` | IPAM CNI Plugin image repository  |
 | `ipamPlugin.version` | string | `v0.5.1-amd64` | IPAM CNI Plugin image version  |
 | `ipamPlugin.imagePullSecrets` | list | `[]` | An optional list of references to secrets to use for pulling any of the IPAM CNI Plugin image |
+
+
+##### Daemonset Node Affinity Override
+
+Can be set to override the default NodeAffinity used by various DaemonSets deployed by network operator, e.g. whereabouts, multus, cni-plugins.
+See [Kubernetes Node Affinity](https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity/)
+
+
+| Name | Type | Default | description |
```

Comment:
nit: you can just add the parameter to the table in `### General parameters`

currently this section is added is under "secondary network" however node affinity also applies to OFED driver container as well

Reply:
Hi @adrianchiris can you please take a look again?